src/db/migrations/init.sql

```diff
@@ -15,6 +15,7 @@ CREATE TABLE IF NOT EXISTS channels (
 
 -- Create messages table.
 CREATE TABLE IF NOT EXISTS messages (
+    rowid INTEGER NOT NULL PRIMARY KEY AUTOINCREMENT,
```

Comment:
Had to add this because otherwise SQLite complains about foreign keys. Put `rowid` out of convenience but we can name it something else (e.g. `id`).

Reply:
Generally I would prefer just a standard "id" yeah, but not really concerned with naming schemes right now. Just focusing on the user experience and getting an MVP off the ground.